src/convert.js

```diff
@@ -19,24 +21,20 @@ function main() {
       .option(
         "--outdir <outdir>",
         "output directory",
-        `${path.join(process.cwd(), "dist")}`
+        `${path.join(process.cwd(), "tmp")}`
```

Comment:
This is also used in npm-scripts so should it come from cli command (with fallback) or at least to those constants in the beginning of the file?

Reply:
Removed defaults for parameters, these are anyways provided in package.json explicitly.